package/gluon-tunneldigger-watchdog/files/usr/bin/tunneldigger-watchdog

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+PIDFILE=/var/run/tunneldigger.mesh-vpn.pid
+
+if [ "$(uci get tunneldigger.@broker[0].enabled)" == "1" ]; then
+    if [ "$(pgrep tunneldigger | head -n 1)" != "$(cat $PIDFILE)" ]; then
```

Comment:
I guess this could be `pidof tunneldigger`? Again, this should probably learn to cope with multiple instances (but this can be done later).